jaeger_client/sampler.py

```diff
@@ -340,11 +394,11 @@ def _init_polling(self):
 
     def _delayed_polling(self):
         periodic = PeriodicCallback(
-            callback=self.poll_sampling_manager,
+            callback=self._poll_sampling_manager,
             # convert interval to milliseconds
             callback_time=self.sampling_refresh_interval * 1000,
             io_loop=self.io_loop)
-        self.poll_sampling_manager()  # Initial sample now
+        self._poll_sampling_manager()  # Initial sample now
```

Comment:
not sure, perhaps it meant "initialize sampler now", as it kicks off the very first refresh of the strategy and then starts periodic polling